asabiris/output/pushnotification/service.py

```
@@ -0,0 +1,107 @@
+
+class PushOutputService(asab.Service):
+	def __init__(self, app, service_name="PushOutputService"):
+		super().__init__(app, service_name)
+		self.Url = asab.Config.get("push", "url", fallback="https://ntfy.sh").strip()
+		self.DefaultTopic = asab.Config.get("push", "default_topic", fallback="").strip()
+		self.Timeout = int(asab.Config.get("push", "timeout", fallback="10"))
+
+		# Optional: tenant config service if you later want per-tenant topics/servers
+		self.ConfigService = app.get_service("TenantConfigExtractionService")
+
+	async def send(self, push_data, tenant=None):
```

Ruff (0.14.2)

Unused method argument: `tenant`

(ARG002)